paper/paper.md

```diff
@@ -43,9 +44,9 @@ The gap that E2EDNA family of programs addresses is the absence of a non-black-b
 
 # Components and Features
 
-With E2EDNA 2.0 our goal is to create an all-python simulation package that is easy to install and use in python-based computational workflows. As shown in Figure\autoref{fig:figure_workflow}, the complete simulation pipeline in E2EDNA 2.0 consists of the following main steps: 1. Secondary structure prediction, 2. Tertiary structure prediction, 3. Molecular dynamics simulation, and 4. Aptamer-ligand docking. The input into E2EDNA 2.0 is the DNA sequence in the FASTA format, the structure of the ligand in the pdb format (optional), and the choice of simulation mode. The output includes secondary structure in dot-bracket notation, tertiary structures in pdb format of free aptamer and aptamer-ligand complex (optional), and simulation trajectories in the dcd format. The analysis of the generated data is not performed by E2EDNA 2.0.
+With E2EDNA 2.0 our goal is to create an all-python simulation package that is easy to install and use in python-based computational workflows. As shown in Figure\autoref{fig:figure_workflow}, the complete simulation pipeline in E2EDNA 2.0 consists of the following main steps: 1. Secondary structure prediction, 2. Tertiary structure prediction, 3. Molecular dynamics simulation, and 4. Aptamer-ligand docking. The key inputs into E2EDNA 2.0 are the DNA sequence in the FASTA format, the structure of the ligand in the pdb format (optional), solvent ionic strength, and the choice of simulation mode. The output includes secondary structure in dot-bracket notation, tertiary structures in pdb format of free aptamer and aptamer-ligand complex (optional), and simulation trajectories in the dcd format. Detailed analysis of the generated trajectories is not performed by E2EDNA 2.0, though is straightforward to set-up for a particular workflow using built-in or user-specified functions.
```

Comment:
1) I thought it might be beneficial to show the flexibility  - that it could adapt to various conditions. If we assume most users will only use 'physiological conditions', we should somewhere else point out the flexibility in this dimension.

2) See my comment above - if we are doing no analysis at all that's fair, but I did want to point out that we make it easy to get started.

Reply:
We should make a note saying that some of these parameters need to be adjusted in the main. I think this should be fine.

Reply:
1. Okay, let's remove the "solvent ionic strength" from the scope of "key inputs" here AND briefly note that "there are plenty of parameters besides 'key inputs' that can be customized in the main.py, such as solvent ionic strength". Then in the documentation, we clarify what parameters (solvent ionic strength, temperature, pH, etc) can be customized. @Lena: we planned to make a similar note in the documentation that it is left to user to modify the FF used in simulations - I will add it the documentation.

Reply:
```suggestion
With E2EDNA 2.0 our goal is to create an all-python simulation package that is easy to install and use in python-based computational workflows. As shown in \autoref{fig:figure_workflow}, the complete simulation pipeline in E2EDNA 2.0 consists of the following main steps: 1. Secondary structure prediction, 2. Tertiary structure prediction, 3. Molecular dynamics simulation, and 4. Aptamer-ligand docking. The key inputs into E2EDNA 2.0 are the DNA sequence in the FASTA format, the structure of the ligand in the pdb format (optional), and the choice of simulation mode. The output includes secondary structure in dot-bracket notation, tertiary structures in pdb format of free aptamer and aptamer-ligand complex (optional), and simulation trajectories in the dcd format. It is worth noting that there are other parameters besides "key inputs" that can be customized in the main.py, such as solvent ionic strength. Detailed analysis of the generated trajectories is not performed by E2EDNA 2.0, though is straightforward to set-up for a particular workflow using built-in or user-specified functions.
```